### PR TITLE
Set `user.microcloud` when bootstrapping LXD

### PR DESCRIPTION
**Pull request by masnax**
CC @edlerd @mas-who

### CODE REVIEW

File: test/suites/upgrade.sh [hunk 1]
```diff
@@ -150,6 +150,7 @@ ovn:
     # Fourth upgrade MicroCloud.
     for m in micro01 micro02 micro03; do
       lxc exec "${m}" -- snap refresh microcloud --channel "${microcloud_target}"
+      set_debug_binaries "${m}"
```
Comment (roosterfish) [inline test/suites/upgrade.sh:153]: Aren't those already copied to the respective location? Even though MicroCloud 1 snap is ignoring them the binaries should still have been copied to the `/var/snap/microcloud/common/` path and AFAIK the `snap refresh` isn't modifying this.

**Review by roosterfish — CHANGES_REQUESTED** [review]
Looks good, only one comment to the test suite.

File: test/suites/upgrade.sh (hunk 1 above)
Comment (roosterfish) [reply test/suites/upgrade.sh:153]: Should get set [here](https://github.com/canonical/microcloud/blob/main/test/includes/microcloud.sh#L1277) already.

Comment (roosterfish) [reply test/suites/upgrade.sh:153]: Yeah good point. Maybe adding a condition in the `setup_system` to only inject if MicroCloud >= 2? Same as with the `dm-crypt` stuff.

Comment (masnax) [reply test/suites/upgrade.sh:153]: Agreed, except instead of >=2 it would be only inject if `/edge`, and skip it `/stable`.

Comment (masnax) [reply test/suites/upgrade.sh:153]: Actually now that I'm thinking about it, silently having cases that skip the debug binary can be very destructive if you forget about it. 

Might just be better to Manually delete the debug binaries in the upgrade tests.

**Comment by masnax** [comment]
I've also switched to use the version string instead of "true" as the value for the `user.microcloud` key.

**Review by roosterfish — APPROVED** [review]

**Pull request merged**